tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -0,0 +1,382 @@
+        """List the info of RHEV Compute Resource
+
+        :id: 1b18f6e8-c431-41ab-ae49-a2bbb74712f2
+
+        :steps:
+
+            1. Display the information of RHEV compute resource.
+
+        :expectedresults: RHEV Compute resource Info is displayed
+
+        :CaseAutomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_delete_by_name(self):
+        """Delete the RHEV compute resource by name
+
+        :id: ac84acbe-3e02-4f49-9695-b668df28b353
+
+        :steps:
+
+            1. Delete the RHEV compute resource with name parameter
+
+        :expectedresults: Compute resource deleted
+
+        :CaseAutomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_delete_by_id(self):
+        """Delete the RHEV compute resource by id
+
+        :id: 4bcd4fa3-df8b-4773-b142-e47458116552
+
+        :steps:
+
+            1. Delete the RHEV compute resource with name id
+
+        :expectedresults: Compute resource deleted
+
+        :CaseAutomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_negative_create_rhev_with_url(self):
+        """RHEV Compute Resource negative create with invalid values
+
+        :id: 1f318a4b-8dca-491b-b56d-cff773ed624e
+
+        :steps:
+
+            1. Create RHEV compute resource with invalid url for url parameter
+
+        :expectedresults: Compute resource not created
+
+        :CaseAutomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_negative_create_with_same_name(self):
+        """RHEV Compute Resource negative create with the same name
+
+        :id: f00813ef-df31-462c-aa87-479b8272aea3
+
+        :steps:
+
+            1. Create a RHEV compute resource.
+            2. Create another RHEV compute resource with same name.
+
+        :expectedresults: Compute resource not created
+
+        :CaseAutomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_update_name(self):
+        """RHEV Compute Resource positive update
+
+        :id: 5ca29b81-d1f0-409f-843d-aa5daf957d7f
+
+        :steps:
+
+            1. Create a RHEV compute resource
+            2. Update the name of the created compute resource
+
+        :expectedresults: Compute Resource successfully updated
+
+        :CaseAutomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier2
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_add_image_rhev_with_name(self):
+        """Add images to the RHEV compute resource
+
+        :id: 6c7f4169-2e78-44d6-87af-434146093bcc
+
+        :setup: Images as templates in rhev.
```

Comment:
Perfect. Then please use `Images/templates should be present in RHEV-M itself, so that satellite can use them.` instead `Images as templates in rhev.`